etl/steps/data/garden/oecd/2023-05-19/co2_air_transport.meta.yml

```diff
@@ -229,9 +106,10 @@ tables:
         title: Monthly CO₂ total emissions from aviation
         unit: tonnes
         short_unit: t
-        description: Includes emissions from both domestict and international passegener flights. International aviation emissions are assigned to the country of departure.
+        description:  Includes emissions from both domestic aviation. International aviation emissions are assigned to the country of departure. CO₂ emissions from commercial passenger flights only.
```

Comment:
Should this still say "domestic and international aviation"?

Reply:
yep! thanks for spotting this :)